src/app/shared/card/card.component.scss

```diff
@@ -0,0 +1,34 @@
+.card {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+  width: 350px;
+  height: 150px;
+  padding: 20px;
+  margin: 5px;
+  border-radius: 30px;
+  border: 2px solid #e4e4e4;
+  background-color: #e4e4e4;
+}
+
+.card:hover {
+  background-color: #ffff;
+  border: 2px solid #ff1e23;
+}
+
+card-title {
```

Comment:
Is this a class? you should add the "." in front of it

Reply:
Fixed